src/lucky_migrator/create_table_statement.cr

```diff
@@ -118,6 +119,41 @@ class LuckyMigrator::CreateTableStatement
     end
   end
 
+  def default_value(type : String.class, default : String, name)
+    if %w[now() current_time current_timestamp].includes?(default.downcase)
+      return " DEFAULT NOW()"
+    end
+    return " DEFAULT '#{default}'"
+  end
+
+  def default_value(type : Time.class, default : (String | Time | Symbol), name)
+    if default.is_a?(Symbol) && default == :now
+      " DEFAULT NOW()"
+    elsif default.is_a?(String) && %w[now() current_timestamp].includes?(default.downcase)
```

Comment:
I think instead of having multiple ways to set the default as the current timestamp, there should be just one way. How about just `:now`? I think that's the most clear. What do you think?

Reply:
Yes that's a good idea. I was trying to catch all possible inputs, but I guess it's better if we have a clean and explicit api. Then it will be easier to document anyway